lib/rules/no-inline-comments.js

```diff
@@ -14,6 +15,13 @@ const astUtils = require("./utils/ast-utils");
 module.exports = {
 	meta: {
 		type: "suggestion",
+		deprecated: {
```

Comment:
When we add `deprecated`, we should remove `frozen: true` (https://github.com/eslint/eslint/pull/19667#discussion_r2063810497).

Reply:
Done!